src/main/kotlin/org/rust/ide/console/RsConsoleRunnerFactory.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.console
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.project.Project
+
+class RsConsoleRunnerFactory {
+
+    fun createConsoleRunner(project: Project, contextModule: Module?): RsConsoleRunner {
+        return RsConsoleRunnerImpl(project, "IntellijRustRepl.rs")
```

Comment:
Let's use `Rust Console` instead of `IntellijRustRepl.rs` here (but still use `IntellijRustRepl.rs` in `org.rust.ide.console.RsConsoleRunnerImpl#createConsoleView`). Also, please extract such names to static fields.